README.md

```diff
@@ -143,7 +121,7 @@
       - [Add Pipeline Source](#add-pipeline-source)
 
 ## General
-_jfrog-client-go_ is a library which provides Go APIs to performs actions on JFrog Artifactory or Bintray from your Go application.
+_jfrog-client-go_ is a library which provides Go APIs to performs actions on JFrog Artifactory from your Go application.
```

Comment:
...on JFrog Artifactory from your... --> ...on JFrog Artifactory, Xray and Distribution from your...